aws/components/dnszone/state.ftl

```diff
@@ -5,76 +5,38 @@
     [#local solution = getOccurrenceSolution(occurrence) ]
     [#local locations = getOccurrenceLocations(occurrence) ]
 
-    [#local privateZone = (solution.Profiles.Network)?has_content]
-
-    [#local resources = {}]
-    [#local attributes = {}]
-    [#local roles = {
-        "Inbound" : {},
-        "Outbound" : {}
-    }]
-
-    [#if privateZone]
-
-        [#local zoneId = formatResourceId(AWS_ROUTE53_HOSTED_ZONE_RESOURCE_TYPE, core.Id) ]
-
-        [#local domainObject = getCertificateObject(
-            {} +
-            attributeIfContent(
-                "Domain",
-                (solution.Domain)!""
-            ) +
-            attributeIfTrue(
-                "IncludeInDomain",
-                solution.IncludeInDomain.Configured
-            )
-        )]
-        [#local domainName = getCertificatePrimaryDomain(domainObject).Name ]
-
-        [#local resources = {
-            "zone" : {
-                "Id": zoneId,
-                "Name" : domainName,
-                "Type": AWS_ROUTE53_HOSTED_ZONE_RESOURCE_TYPE
-            }
-        }]
-
-        [#local attributes = {
-            "DOMAIN" : domainName,
-            "ZONE" : getExistingReference(zoneId)
-        }]
-
-    [#else]
```

Comment:
The ability to manually configure the zone from external info seems to have been removed. This wasn't mentioned in the description,

The aim of this was to permit a pre-existing zone to be imported.